Formula/geant4.rb

```diff
@@ -1,9 +1,9 @@
 class Geant4 < Formula
   desc "Simulation toolkit for particle transport through matter"
   homepage "https://geant4.web.cern.ch"
-  url "https://geant4-data.web.cern.ch/geant4-data/releases/source/geant4.10.06.p02.tar.gz"
-  version "10.6.2"
-  sha256 "ecdadbf846807af8baa071f38104fb0dcc24847c8475cd8397302e2aefa8f66f"
+  url "https://geant4-data.web.cern.ch/geant4-data/releases/source/geant4.10.07.tar.gz"
+  version "10.7.0"
+  sha256 "776ea45230d26fffebf0cf7a342af5131135759a0f70e1b4a1a401f1d1eaad4a"
```

Comment:
You should add a license identifier, like `license "LGPL-2.1-or-later"` (I think there is a list of identifiers somewhere, or it's a standard, I also don't know which license Geant is under, it's one of those that doesn't self identify.

Reply:
List of identifiers here: https://spdx.org/licenses/

Reply:
Yep, that's it! I just don't know what one [this](https://github.com/Geant4/geant4/blob/master/LICENSE) is. :)

Reply:
Well, there's always `:cannot_represent`, but that's only for if you really need it.

Reply:
I've added as `:cannot_represent` as there's no SPDX identifier (unsurprisingly!) and discussions here and in the main repo seemed to indicate that's a reasonable marker for cases like this?

Reply:
That license actually looks like it's non-free and debian seems to agree: https://wiki.debian.org/DebianScience/Geant4 so that would mean we'll have to remove the formula.

Reply:
Rough. Though, having formulae moved to an external tap is no longer as bad an experience for an end-user as it used to be: https://brew.sh/2020/11/18/homebrew-tap-with-bottles-uploaded-to-github-releases/

Setting up a tap for Geant4 this way should make it almost as easy to install as if it were still in Homebrew/core.

Reply:
Sure, can look at that now it's much easier to set up. Geant4 would still depend on `homebrew-core` packages, so if there are any tips/pointers on managing that (e.g. tracking version/revision bumps to formulae like `clhep`, `qt`), let me know.

Just to confirm for my info going forward, _any_ license with SPDX entry will be ok going forward, or must it be one from under the DFSG?

Reply:
It's been DFSG for a while, the identifier for licenses is new though. So we don't have all formulae mapped yet.

https://docs.brew.sh/Acceptable-Formulae#we-dont-like-binary-formulae